cmd/release/README.md

```diff
@@ -0,0 +1,47 @@
+# Release
+
+**Release** is a command line tool that automates the versioning and releasing of multi-module Go projects.
+
+## Limitations
+
+- All modules have the same version.
+- Git tags are treated as the source of truth for the version.
```

Comment:
How's this a problem?

Reply:
I have seen some tools using some kind of a file to declare all modules and respective versions. The tool then uses this file to create tags etc. The downside of this approach is that it requires manually updating this file for every release.

My goal with this tool is to make it fully automated using only git data as source of truth